templates/compositions/ec-component-content-item-block/content-item-block.html.twig

```diff
@@ -0,0 +1,93 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content Item Block.
+   */
+  Parameters:
+    - "content_item_block" (associative array) (default: predefined structure): format:
+        {
+          type: '', (string) (default: 'one'): Type of Content Item Block
+          title: '', (string) (default: ''): Title of Content Item Block
+          items: [], (array) (default: []): Content Item's for the Content Item Block
+          button: {}, (associative array) (default: {}): Predefined structure compatible with Link component.
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _content_item_block = {
+  type: 'one',
+  title: '',
+  items: [],
+  button: {}
+} %}
+
+{% if content_item_block is defined %}
+  {% set _content_item_block = _content_item_block|merge(content_item_block) %}
+{% endif %}
+
+{% set _css_class = 'ecl-content-item-block ecl-container' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
```

Comment:
Only this is enough:
```
 {% if extra_classes %}
```
this Twig code will be compiled in the right PHP code, let's remove them all, it clutters the code.

Reply:
ok :+1: